front/app/components/UI/Error/index.tsx

```diff
@@ -172,6 +172,21 @@ export type TFieldName =
   | 'category_name'
   | 'nav_bar_item_title_multiloc';
 
+export const findErrorMessage = (
+  fieldName: TFieldName | undefined,
+  error: string
+) => {
+  if (fieldName && messages[`${fieldName}_${error}`]) {
+    return messages[`${fieldName}_${error}`] as Message;
```

Comment:
I think in a previous CDM we talked about not using string interpolation anymore to get messages by name... I can't find a style guide or anything but I think it's good practice.

Reply:
Hm, I didn't really change this method - just took it out of the component to make it reusable for the `Feedback`. But happy to improve it. What would be the suggested alternative?

Reply:
Good suggestion but it goes out of scope for this PR - added a [separate ticket](https://citizenlab.atlassian.net/browse/TEC-204) so we can address it in the future.